app/models/manageiq/providers/openstack/cloud_manager.rb

```diff
@@ -31,6 +31,8 @@ class ManageIQ::Providers::Openstack::CloudManager < EmsCloud
 
   include ManageIQ::Providers::Openstack::ManagerMixin
 
+  validates :authentications, :presence => {:message => 'You must enter credentials.'}
```

Comment:
I'm fine with using `base_manager`... how would a subclass disable such a validation?  Via a flag at `base_manager` that a subclass could override as false?